client/a8c-for-agencies/components/a4a-request-wp-admin-access/index.tsx

```diff
@@ -29,10 +29,6 @@ export function A4ARequestWPAdminAccess() {
 						'Ask the Agency owner to provide you WP-Admin access to this site in order to manage its plugins and features.'
 					) }
 				</div>
-				<div className="a4a-request-wp-admin-access__learn-more">
```

Comment:
Should we remove the `a4a-request-wp-admin-access__learn-more` from the style file? It's not used anymore.

Reply:
Good catch! this has been fixed.